src/OnlySpans.PolyLeads.Api/Services/LLM/YandexGptClient.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Net.Http.Headers;
+using Microsoft.Extensions.Options;
+using OnlySpans.PolyLeads.Api.Abstractions.LLM;
+using OnlySpans.PolyLeads.Api.Data.Options;
+using OnlySpans.PolyLeads.Api.Data.Records.LLM;
+
+namespace OnlySpans.PolyLeads.Api.Services.LLM;
+
+public sealed class YandexGptClient : ILLMClient
+{
+    private readonly HttpClient _httpClient;
+    
+    private readonly IOptions<LLMOptions> _options;
```

Comment:
```suggestion
    private readonly LLMOptions>_options;
```

в конструкторе просто `.Value` сразу запишем